pull-request.py

```diff
@@ -73,20 +73,15 @@ def create_pull_request(
 ):
 
     # Check if the branch already has a pull request open
+    params = {"base": target, "head": source, "state": "open"}
     data = {"base": target, "head": source, "body": body}
-    print("Data for checking if pull request exists: %s" % data)
-    response = requests.get(PULLS_URL, json=data)
+    print("Params for checking if pull request exists: %s" % params)
+    response = requests.get(PULLS_URL, params=params)
```

Comment:
Is there any reason to use params over json? As I understand it, params will provide the dict as GET parameters (e.g.., akin to putting them in the url), data will provide the dict in the request body, and json= (the current) does the same but includes that the content type is application/json. Is it more standard to provide them as GET parameters over in the body here?

Reply:
I _believe_ more standard. As a clue, in the github docs (https://developer.github.com/v3/pulls/) they are called "Parameters" for the GET request, and for POST, PUT, PATCH or whatever github calls them "Inputs". However, I don't know for sure that github's API won't also fall back to looking in the body (or json, like you said) if it doesn't see any url params, so it might work either way.